.github/workflows/rerun-workflows.yml

```diff
@@ -0,0 +1,31 @@
+name: Rerun Flaky Workflows
+
+on:
+  workflow_run:
+    workflows: [Backend, Driver Tests, E2E Tests, Frontend]
```

Comment:
1. This is somewhat fragile because it depends on the name of the workflow that we can change
2. These seem to be the most fragile workflows. We can add or remove some as needed.